community/server/src/main/java/org/neo4j/server/ServerCommandLineArgs.java

```diff
@@ -43,6 +43,7 @@
 public class ServerCommandLineArgs
```

Comment:
Do you think you could fix the javadoc here while you're at it?

* `--config` is never used.
* `-C` is never used
* `--config-dir` is not documented
* `--home-dir` is not documented